src/vcpkg/metrics.cpp

```diff
@@ -88,6 +88,7 @@ namespace vcpkg
     }};
 
     const constexpr std::array<BoolMetricEntry, static_cast<size_t>(BoolMetric::COUNT)> all_bool_metrics{{
+        {BoolMetric::DetectedContainerEnvironment, "detected_container_environment"},
```

Comment:
```suggestion
        {BoolMetric::DetectedContainer, "detected_container"},
```

Also should this be a define metric?